internal/consensus/pbts_test.go

```diff
@@ -1,56 +1,299 @@
 package consensus
 
 import (
+	"bytes"
+	"context"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/tendermint/tendermint/abci/example/kvstore"
+	tmpubsub "github.com/tendermint/tendermint/libs/pubsub"
 	tmtimemocks "github.com/tendermint/tendermint/libs/time/mocks"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 	"github.com/tendermint/tendermint/types"
 )
 
+type pbtsTestHarness struct {
```

Comment:
Although all of this stuff is unexported, it's doing enough interesting work that I think it deserves some comments—on the type, and on the fields maybe to say what they're supposed to be for.